lib/auth/auth_with_roles.go

```diff
@@ -5179,6 +5179,19 @@ func (a *ServerWithRoles) MaintainSessionPresence(ctx context.Context) (proto.Au
 	return nil, trace.NotImplemented(notImplementedMessage)
 }
 
+// SubmitUsageEvent submits an external usage event.
+func (a *ServerWithRoles) SubmitUsageEvent(ctx context.Context, req *proto.SubmitUsageEventRequest) error {
+	if err := a.action(apidefaults.Namespace, types.KindUsageEvent, types.VerbCreate); err != nil {
```

Comment:
Where do we ensure that users or system roles have create access for usage events?

Edit: I see it's in the default implicit role.

Reply:
Yup, kind of a no-op, I think the only entities that can't submit events are nodes? Frankly I'm still tempted to remove the RBAC check here outright, but that feels wrong for other reasons.